.gitlab-ci.yml

```diff
@@ -127,11 +136,28 @@ Build -tests RPM for RHEL:
     matrix:
       - <<: *RPM_RUNNERS_RHEL
 
+RPM for RHEL-GA:
+  stage: rpmbuild
+  extends: .terraform
+  rules:
+    - !reference [.nightly_rules_ga/eus, rules]
+  script:
+    - sh "schutzbot/mockbuild.sh"
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/rhel-8.4-ga-x86_64
+          - aws/rhel-8.4-ga-aarch64
+          - aws/rhel-8.9-ga-x86_64
+          - aws/rhel-8.9-ga-aarch64
+          - aws/rhel-9.3-ga-x86_64
+          - aws/rhel-9.3-ga-aarch64
+
```

Comment:
I don't think we need a new GA rpm build, we can just use the regular one with modified rules.

Reply:
@atodorov Why do we even build the rpms for GA? In case we need to do a z-stream release with some bug so to make sure the latest rpm from main still works on the given GA? Would we even ship such a new rpm? I suppose we should rather make a fix on the respective GA branch no? Should we maybe do the same as for nightly and that is use the package rpms that are shipped there and rebuild the `-tests` package only?

Reply:
@jrusz I named it nightly simply because of the fact they will run at night, but that can be ofc changed. I overlooked the rules not working for some of the jobs since in a few cases, they are overriden directly inside the jobs. Will investigate and fix that.

Reply:
The reason they run at night is because it's expected to have the least impact on daily productivity and also to have the results in the morning. The main reason behind the name is because it's running on nightly composes.

Reply:
> Why do we even build the rpms for GA? 

These are RPMs built from the software under test, regardless of which runner OS it is using.  My understanding is that with these new pipelines we're testing "latest osbuild-composer on already released distros". So you need to build them, no? 

> Would we even ship such a new rpm? 

Yes, it will go out as an advisory. 

> I suppose we should rather make a fix on the respective GA branch no? 

I think the fix should land first into `main` and then be cherry-picked into the relevant downstream branches. However this opens the question of what's the development model here and where do we want to be testing stuff. 

OTOH there is still value in knowing that latest osbuild-composer releases can be dropped into an older distro version. 

I think unltimately we'll need testing both ways but that's a bigger discussion than a GH PR review comment I think.